improver_tests/acceptance/SHA256SUMS

```diff
@@ -633,73 +633,73 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
-2e6d8a6ed2b7e4ae88c74b25ad04dc38d7cce58574a0c52339bf3076fe0b8929  ./wxcode-modal/spot_ties/20201209T1100Z-weather_symbols-PT01H.nc
-5bfc0319d35a7b7a230fdd51f3e8b6bfddf5632b32dde8e0ff023a6b106268f7  ./wxcode-modal/spot_ties/20201209T1200Z-weather_symbols-PT01H.nc
-8f94e92886c8df8070db44b273037b032615e4b8768a3e45c708686061f0c8bc  ./wxcode-modal/spot_ties/20201209T1300Z-weather_symbols-PT01H.nc
-4fb839e28d711fd0ebbf87090358644b62f7dfa0d8dcdcad9c5dc322ce28eaac  ./wxcode-modal/spot_ties/20201209T1400Z-weather_symbols-PT01H.nc
-c798e3530eac9a45305711d5e91daa56d5005b826ce85bb48e2889e328ea965c  ./wxcode-modal/spot_ties/20201209T1500Z-weather_symbols-PT01H.nc
-83ba8460875232fc555976872cdb5ac59d5092ccc0da88f88beb9168a3403ac3  ./wxcode-modal/spot_ties/20201209T1600Z-weather_symbols-PT01H.nc
-a6698b2aae8bc223a2d362ccb3047dd9882e6c0df21962e55c577e1ad5850e29  ./wxcode-modal/spot_ties/20201209T1700Z-weather_symbols-PT01H.nc
-9290b2899eedfca038226c6707e100025ce68b0798972a60360845ccf36858d2  ./wxcode-modal/spot_ties/20201209T1800Z-weather_symbols-PT01H.nc
-ceb7106e5b8919590327a75d4a8b740fb1a2c7cb5a271a5bf980f0c1396c7d22  ./wxcode-modal/spot_ties/kgo.nc
+6513b722e1fdca6ad68acd768f3506ec814f78e9ccd4799972ec00a2316ef7cf  ./wxcode-modal/blend_mismatch_inputs/20201209T0700Z-weather_symbols-PT01H.nc
+d4c5d626ad2e5a805f90f2902f6da5d33651eb6fdcb1e16116e2290d0c608bf6  ./wxcode-modal/blend_mismatch_inputs/20201209T0800Z-weather_symbols-PT01H.nc
+4642fc43faf4bdde6a678578868fdebb22163064fe6ef0eef41f2f39a959ca89  ./wxcode-modal/blend_mismatch_inputs/20201209T0900Z-weather_symbols-PT01H.nc
+ce6a854694aecd515063c91774eeb6160f39c9dbf5e970f72f960a3527e157e9  ./wxcode-modal/blend_mismatch_inputs/20201209T1000Z-weather_symbols-PT01H.nc
+300c38e7f522ff1e9257708b183148f3441bb1a861c15eaff2a5f1713bbbc872  ./wxcode-modal/blend_mismatch_inputs/20201209T1100Z-weather_symbols-PT01H.nc
+df077177d1ac4b4faccb945f5e64c0bcf2bf19356dee5acef8caffc5db602823  ./wxcode-modal/blend_mismatch_inputs/20201209T1200Z-weather_symbols-PT01H.nc
+5ab6b249ec615ddab9995629aaddbd85bba352ad52b7bcb0be5b3037d75c22ca  ./wxcode-modal/blend_mismatch_inputs/20201209T1300Z-weather_symbols-PT01H.nc
+340205e12625ff9466c1afba8c007343aa40e21b837ab2f2143fa032e6c47bd3  ./wxcode-modal/blend_mismatch_inputs/20201209T1400Z-weather_symbols-PT01H.nc
+776739d288b6e43b3d9341ba301feead160986d2540514f0663f243ca003e55a  ./wxcode-modal/blend_mismatch_inputs/20201209T1500Z-weather_symbols-PT01H.nc
+4655358c22abdd0acfd5f4e08f156a7f4f629a1764dd8314f28f121964725078  ./wxcode-modal/blend_mismatch_inputs/20201209T1600Z-weather_symbols-PT01H.nc
+28ee3f9d419bb9766cc68e1c74fa9e8e5010dc63a600635a54356a33795f6644  ./wxcode-modal/blend_mismatch_inputs/20201209T1700Z-weather_symbols-PT01H.nc
+073fd0175e35afcf7f9fe61fedfa63bbd9ea5299f68d8b0287bc30085339b942  ./wxcode-modal/blend_mismatch_inputs/20201209T1800Z-weather_symbols-PT01H.nc
+a6b3f48f8c2003c2d84da0a4cb8634bed24f0d54594d6cc9235c470858f49c19  ./wxcode-modal/blend_mismatch_inputs/kgo.nc
```

Comment:
I think that the known good outputs have `wmo_id` and `met_office_site_id` coordinates with `unknown` units, rather than `no_unit` as it seems like even though `no_unit` is in the cube prior to saving, no units are actually saved for e.g. the `wmo_id` coordinate (as seen from `ncdump -h`). Therefore loading the saved known good outputs results in the `wmo_id` and `met_office_site_id` coordinates having an `unknown` unit. 

I'm not quite sure why the units for the `wmo_id` and `met_office_site_id` coordinates aren't being saved in kgo, given that they're present in the input cubes. Should there be any concern about this?